src/core/load-fragment-gperf.gperf.m4

```diff
@@ -161,6 +161,7 @@ $1.StartupCPUWeight,             config_parse_cg_weight,             0,
 $1.CPUShares,                    config_parse_cpu_shares,            0,                             offsetof($1, cgroup_context.cpu_shares)
 $1.StartupCPUShares,             config_parse_cpu_shares,            0,                             offsetof($1, cgroup_context.startup_cpu_shares)
 $1.CPUQuota,                     config_parse_cpu_quota,             0,                             offsetof($1, cgroup_context)
+$1.CPUQuotaPeriod,               config_parse_cpu_quota_period,      0,                             offsetof($1, cgroup_context)
```

Comment:
hmm, in light of the other discussions this should probably be called CPUQuotaPeriodSec= or so